woocommerce/woocommerce-android
# Add simulated card reader toggle logic

Closes: #7536 


### Description


This PR adds the logic to enable/disable the Simulated reader using the toggle button in the Developer Options screen 

### Testing instructions


1 - in Debug build -> Go to the App Menu - Settings - Developer Options
2 - Toggle the Simulated Reader ON
3 - create a simple payment or an order
4 - take the payment via credit card
5 - make sure the simulated reader is activated (It might ask you to select which reader, select the one according to where your store is based - US or Canada)
6 - Go back to the settings screen - Developer Options
7 - Toggle the simulated reader OFF
8 - repeat the steps to collect a payment
9 - Make sure the simulated reader won't activate

### Images/gif


https://user-images.githubusercontent.com/30724184/198228337-3d90eb06-31f6-40e8-a0d1-d733e2661687.mp4

Please note: Currently, untoggling the button is not clearing the selected card reader  (LAST_CONNECTED_CARD_READER_ID) so the button is not disabling the simulated reader. 

This will be addressed in the next PR
